cms/search/utils.py

```diff
@@ -0,0 +1,70 @@
+from typing import TYPE_CHECKING
+
+from django.apps import apps
+from django.utils.encoding import force_str
+from wagtail.rich_text import get_text_for_indexing
+
+from cms.release_calendar.enums import ReleaseStatus
+
+if TYPE_CHECKING:
+    from wagtail.models import Page
+
+
+def build_standard_resource_dict(page: "Page") -> dict:
+    """Returns a dict with the standard resource fields.
+    This covers the non-release case (and also forms the base of the release case).
+    """
+    return {
+        "uri": page.url_path.removeprefix("/home"),
```

Comment:
I would not rely on the prefix being `/home/`

Instead:
```pycon
>>> s = "/home/releases/psf-december-2024-release/"
>>> s.strip("/").split("/", 1)
['home', 'releases/psf-december-2024-release']
>>> s.strip("/").split("/", 1)[-1]
'releases/psf-december-2024-release'
>>> "/home/".strip("/").split("/", 1)[-1]
'home'
```

so:

```suggestion
    path = page.url_path.strip("/").split("/", 1)[-1]

    return {
        "uri": f"/{path}" if not getattr(settings, "WAGTAIL_APPEND_SLASH", True) else f"/{path}/",
```

with a `from django.conf import settings` import